### DISCUSSION TITLE
v4.x 官网中的主题编辑器相关代码示例有吗

### DISCUSSION THREAD

**Discussion by nailuox** [vueComponent/ant-design-vue#6838] [Q&A]
v4.x 官网中的主题编辑器相关代码示例有吗

**Comment by rayadaschn** [comment]
有的，在[ConfigProvider](https://antdv.com/docs/vue/customize-theme-cn#%E4%BF%AE%E6%94%B9%E4%B8%BB%E9%A2%98%E5%8F%98%E9%87%8F) 组件中可修改
